src/sass/modules/_m-schemaform.scss

```diff
@@ -281,4 +281,40 @@ label {
 .schemaform-file-list {
   list-style: none;
   padding-left: 0;
+  > li {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+  > .schemaform-file-item-edit {
+    flex-direction: column;
+    align-items: left;
+  }
+}
+
+.schemaform-file-list-buttons-editing {
+  display: flex;
+  > .schemaform-file-remove-button {
+    width: inherit;
```

Comment:
Is this overridden somewhere else? (That is, do you _have_ to specify an inherited with?)

I assume so, but figured I'd flag it just in case.